processor/src/main/java/com/linecorp/decaton/processor/runtime/ConsumerSupplier.java

```diff
@@ -44,11 +47,14 @@ public ConsumerSupplier(Properties config) {
 
     @Override
     public Consumer<String, byte[]> get() {
-        return new KafkaConsumer<>(mergedProps(), new StringDeserializer(), new ByteArrayDeserializer());
+        return new KafkaConsumer<>(mergedProps(), null, new ByteArrayDeserializer());
```

Comment:
This is bit tricky and dangerous bypass of type check as in we cannot ensure the deserializer specified in properties actually returns `String`.
If it wasn't, I'm not sure at which place does the first error gets thrown, that might makes users confused what has went wrong.
Would it better to take key deserializer (`Function<byte[], String>`) at `ProcessorBuilder` explicitly?

Reply:
I see, added a parameter `keyDeserializer` to `ProcessorsBuilder.consuming()`